charts/super-agent-deployment/values.yaml

```diff
@@ -0,0 +1,103 @@
+# -- Override the name of the chart
+nameOverride: ""
+# -- Override the full name of the release
+fullnameOverride: ""
+
+# -- Name of the Kubernetes cluster monitored. Can be configured also with `global.cluster`.
+cluster: ""
```

Comment:
Is this used?

Reply:
Marked as TODO [here](https://github.com/newrelic/helm-charts/pull/1155#discussion_r1365584627).